Formula/pike.rb

```diff
@@ -56,10 +55,6 @@ def install
       args << "--with-abi=32"
     end
 
-    if build.without? "machine-code"
```

Comment:
If this option was removed then `option "with-machine-code", "Enables machine code"` should be removed above, too.